shtdlib.sh

```diff
@@ -801,14 +801,14 @@ function init_nss_wrapper {
     group_name_pattern="^${TMP_GROUP}"':x:.*:.*$'
     group_pattern='^.*:x:'"${BGID}"':.*$'
 
-    if grep --extended-regexp --quiet "${passwd_name_pattern}" "${tmp_passwd_file}" \
```

Comment:
Any reason why you want to use short options rather than the long, are the long not supported on Alpine?

Reply:
Yeah thats right, this was causing error on the postgres container

Reply:
That's annoying